### PR TITLE
Updater tests

### PR DESCRIPTION
**Pull request by megandouli**
Updater functionality recently changed to sending searchNEU a POST request with updated course/section info. This PR tests most of the changes except for making the request.

### CODE REVIEW

File: tests/updater.test.seq.ts [hunk 1]
```diff
@@ -1,22 +1,164 @@
...
 
+import Updater from '../updater';
 import { Course as CourseType, Section as SectionType, Requisite } from '../types';
 import prisma from '../prisma';
-import Updater, { Notification } from '../updater';
 import Keys from '../Keys';
 import dumpProcessor from '../dumpProcessor';
 import termParser from '../scrapers/classes/parsersxe/termParser';
 
+const SEM_TO_UPDATE = '202130';
```
Comment (gamburgm) [inline tests/updater.test.seq.ts:12]: Do either of these constants exist elsewhere in the codebase to import? If not this is fine.

Comment (gamburgm) [inline tests/updater.test.seq.ts:12]: Do either of these comments exist elsewhere in the codebase? If not that's fine.

File: tests/updater.test.seq.ts [hunk 2]
```diff
@@ -25,381 +167,234 @@ afterEach(async () => {
...
-    ...defaultClassProps,
-    ...defaultSectionProps,
-  };
-
-  // TODO this is low priority
-  describe('modelToUserHash', () => {
-    it('works for followed courses', () => {
+  it('scrapes the right terms to update', async () => {
+    const mockTermParser = jest.fn(async () => { return []; });
+    jest.spyOn(termParser, 'parseSections').mockImplementation(mockTermParser);
```
Comment (gamburgm) [inline tests/updater.test.seq.ts:191]: Could you move as much just spying/mocking into a `beforeAll` or `beforeEach` hook either at the top of the test suite or at the top of the file?

File: tests/updater.test.seq.ts [hunk 3]
```diff
@@ -1,22 +1,164 @@
-import { InputJsonValue } from '@prisma/client';
-import pMap from 'p-map';
+
```
Comment (gamburgm) [inline tests/updater.test.seq.ts:1]: Could you delete this empty line :^)

**Review by gamburgm — CHANGES_REQUESTED** [review]
Only one real comment, overall looks great

File: tests/updater.test.seq.ts (hunk 1 above)
Comment (megandouli) [reply tests/updater.test.seq.ts:12]: do you mean `SEM_TO_UPDATE` and `EMPTY_REQ`?

Comment (megandouli) [reply tests/updater.test.seq.ts:10]: `SEM_TO_UPDATE` is just a constant in the updater atm and `EMPTY_REQ` is a relic from the old updater tests.

**Review by gamburgm — APPROVED** [review]
Awesome!

**Pull request merged**